src/mbtest/server.py

```diff
@@ -152,6 +153,26 @@ def add_imposters(self, definition: Union[Imposter, Iterable[Imposter]]) -> None
             definition.attach(self.host, post.json()["port"], self.server_url)
             self._running_imposters.append(definition)
 
+    def add_stabs(self, definition: Union[Stub, Iterable[Stub]], port: int):
```

Comment:
`add_stubs()` I imagine this should be.

Reply:
Oh. Of course. I'll fix it with pre-commit.